Marlin/src/module/probe.cpp

```diff
@@ -966,7 +959,7 @@ float Probe::probe_at_point(const_float_t rx, const_float_t ry, const ProbePtRai
     switch (raise_type) {
       default: break;
       case PROBE_PT_RAISE:
-        do_z_clearance(Z_CLEARANCE_BETWEEN_PROBES);
+        do_z_clearance_by(Z_CLEARANCE_BETWEEN_PROBES); // Always raise over the current Z position when the probe triggers
```

Comment:
Consider the case where the probe triggers at a lower point (for some reason). That will actually lead to less of a raise.

Reply:
What about where the probe triggers at a higher point? This change leads to no raise. And a bunch of additional raises during G29 variants.

so something like `_MAX(Z_CLEARANCE_BETWEEN_PROBES, Z_CLEARANCE_BETWEEN_PROBES + current_position.z)` then.

Reply:
e19a6d4 implements the above mentioned change, which results in the elimination of extra lifts during G29 w/ HS mode as a result of 98ec0be. Future changes to `Z_PROBE_SAFE_CLEARANCE` shouldn't cause the extra lifts to reoccur now.
(extra lifts in this case meaning performing two z lifts instead of a single one to the same height)